### ISSUE TITLE
Skewed Textures

### ISSUE THREAD

**Issue by bz-next** [BZFlag-Dev/bzflag#346]
https://github.com/BZFlag-Dev/bzflag/issues/315

I still occasionally experience this bug on 2.4.26.

When I start playing, the textures seem OK, but after some time, skewed textures can appear.
I notice the issue mostly on the doors of LouMan's churchyard, and some of the buildings on GEP's Quickshot map.

Client version:
BZFlag client 2.4.26.20230118-MAINT-linux-gnu-SDL2 (protocol 0221) http://BZFlag.org/

**Comment by blast007** [comment]
Could you provide some additional information?
* Graphics card
* Linux distribution and version
* Linux kernel version

**Comment by bz-next** [comment]
Sure. Also, I was able to reproduce the bug today.

I joined a few servers and saw nothing amiss. Then I joined Churchyard R2, and still, everything looked good.

I then switched to windowed mode, and suddenly the textures were skewed. I usually play in windowed, though my client boots in fullscreen, so there's usually a fullscreen->windowed transition during a run of the game for me, which would explain why I've seen this happen many times, but not always.

The problem disappears if I rejoin the server. If I switch back to fullscreen, the problem reappears. So it seems to be related to the fullscreen->windowed transition, regardless of which direction it goes.

![image](https://github.com/BZFlag-Dev/bzflag/assets/158550183/3e937cef-1201-4edb-b126-d0b1847af0e0)

GPU: NVidia GTX 1080 Ti
Linux: Fedora 39
Kernel: Linux fedora 6.7.4-200.fc39.x86_64
